packages/vite/src/utils/tsnode-register.ts

```diff
@@ -0,0 +1,20 @@
+import { workspaceRoot } from '@nrwl/devkit';
+import { resolve } from 'path';
+
+export function tsNodeRegister() {
+  require('ts-node').register({
```

Comment:
One thing we had to do for webpack is to prevent double-registering `ts-node`. This might happen if two different executors calls the register function (e.g. cypress executor calls dev-server, and both register `ts-node`).

This is the guard for that:

```
export function isRegistered() {
  return (
    require.extensions['.ts'] != undefined ||
    require.extensions['.tsx'] != undefined
  );
}
```

Reply:
Done!